includes/ms_builtin.h

```diff
@@ -34,10 +37,8 @@ typedef enum e_exit_argument
 int		ft_echo(char *const *command);
 int		ft_exit(char *const *command, t_params *params);
 
-bool	ft_legal_number(const char *str, long *result);
+bool	is_legal_number(const char *str, long *result);
```

Comment:
```suggestion
bool	legal_number(const char *str, long *result);
```
is_legal_number()だとlegal_numberの判定のみ、参照経由で値を返す役割が見えにくそうな気がしました。
（libftのatoiはis_atoiか？みたいな...？）
本家と重複しますが、legal_number() だけでも良いかもですね

Reply:
相談の結果 str_to_legal_number へ！

Reply:
✅